bindata/bootkube/config/config-overrides.yaml

```diff
@@ -1,12 +1,5 @@
 apiVersion: kubecontrolplane.config.openshift.io/v1
 kind: KubeAPIServerConfig
-aggregatorConfig:
-  proxyClientInfo:
-    certFile: /var/run/secrets/aggregator-client/tls.crt
-    keyFile: /var/run/secrets/aggregator-client/tls.key
```

Comment:
the operator delays startup until it has added these "in code"?

Reply:
> the operator delays startup until it has added these "in code"?

set here and since it is required for a revision, the installer won't start without it.